### DISCUSSION TITLE
Unable to get local issuer certificate even DB_SSL__CA is correctly provided on Kubernetes

### DISCUSSION THREAD

**Discussion by Castafiore** [directus/directus#7155] [Archive]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

I plan to deploy Directus latest version to Kubernetes with Docker. I have set all DB Env Vars to connect Azure Mysql which requires SSL connection. During the initialisation, I get `unable to get local issuer certificate` error, but I can be sure the `DB_SSL__CA` is correctly set (the content of the ca cert file), and the `DB_SSL__REJECT_UNAUTHORIZED` is set to true. Could you help me how to figure it out?

### To Reproduce

1. 

### What version of Directus are you using?

9.0.0-rc.88

### What version of Node.js are you using?

14.15.1

### What database are you using?

Mysql 8.0.15

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying Directus?

Kubernetes & Docker

**Comment by rijkvanzanten** [comment]
Seems like this is a finicky thing in Azure. There's 42.000 results matching the search 'azure "unable to get local issuer certificate"' on Google 😅 

One result mentioned that the CA certificate might've had the wrong line breaks? Is the CA cert in the correct format?

**Reply by Castafiore** [reply]
The CA certificate do have line breaks but is in the correct format below
```
 -----BEGIN CERTIFICATE-----
key
-----END CERTIFICATE-----
```
But I can connect to it with the same CA cert via TablePlus. Besides, even if the CA cert is wrong, the error message `unable to get local issuer certificate` is weird, sound like it wrongly thought the provided CA cert is a local file path, but it is content, so I don't know why. Have you ever met or heard about this kind of issue with DB_SSL__CA?

**Reply by rijkvanzanten** [reply]
No clue, never seen this before. Seems very similar to https://stackoverflow.com/questions/49555307/aks-mysql-ssl-problems-nodejs-knex though

**Reply by Castafiore** [reply]
Thanks a lot! It solved my issue.

**Comment by Castafiore** [comment]
Thanks, the https://stackoverflow.com/questions/49555307/aks-mysql-ssl-problems-nodejs-knex solved my issue.
